setup/pi/setup-teslausb

```diff
@@ -921,6 +921,13 @@ make_root_fs_readonly
 
 upgrade_packages
 
+if [ -n "${INSTALL_USER_REQUESTED_PACKAGES:-}" ]
+then
+  export INSTALL_USER_REQUESTED_PACKAGES
+  copy_script setup/pi/install-user-requested-packages.sh /tmp
+  /tmp/install-user-requested-packages.sh
```

Comment:
I don't think it's worth having a whole separate install script just to call `apt-get`.
Just do it inline here.

Reply:
I was on the fence about this one, I almost made this change prior to your review. I'm out of the country for another week or so, when I return I'll get this fixed up, thank you.